packages/core/src/editor/boxes/GridBox.ts

```diff
@@ -10,7 +10,7 @@ export type GridOrientation = "neutral" | "row" | "column";
 export class GridBox extends Box {
     readonly kind = "GridBox";
     cells: GridCellBox[] = [];
-    private $children: Box[] = []; // TODO find out why this prop is needed
+    private $children: Box[] = []; // TODO question: why is this prop needed?
```

Comment:
I think this property is not needed,  the get children function can just return the "cells". property.

Reply:
DONE